ui/helpers/url.js

```diff
@@ -1,4 +1,3 @@
-import taskcluster from './taskcluster';
```

Comment:
This removal should help fix at least one of the import cycles in bug 1493833 :-)